client/state/connected-applications/schema.js

```diff
@@ -0,0 +1,45 @@
+export default {
+	type: 'array',
+	items: {
+		type: 'object',
+		properties: {
+			ID: { type: 'string' },
```

Comment:
Seeing this now, I wish we'd transformed the interger strings to numbers 😞 https://github.com/Automattic/wp-calypso/pull/24363/files#r187609390

On the other hand, there's no application logic that depends on these being integers. The API could provide integers, I wonder why it doesn't 🤔